src/ppx/geometry.cpp

```diff
@@ -247,6 +260,39 @@ class VertexDataProcessorInterleaved : public VertexDataProcessorBase
         return GetVertexBufferElementCount(pGeom, kBufferIndex);
     }
 
+    virtual uint32_t AppendVertexData(Geometry* pGeom, const TriMeshVertexData16& vtx) override
+    {
+        uint32_t       startSize = GetVertexBufferSize(pGeom, kBufferIndex);
```

Comment:
can this code block within the function being templated instead of a copy of existing code?

Reply:
Thanks for the feedback. Updated.